mailscanner/detail.php

```diff
@@ -63,40 +63,40 @@
 // The sql command to pull the data
 $sql = "
  SELECT
-  DATE_FORMAT(timestamp, '" . DATE_FORMAT . " " . TIME_FORMAT . "') AS 'Received on:',
-  hostname AS 'Received by:',
-  clientip AS 'Received from:',
-  headers 'Received Via:',
+  date_format(timestamp, '" . date_format . " " . time_format . "') as '" . $lang['receivedon04'] . ":',
+  hostname as '" . $lang['receivedby04'] . ":',
+  clientip as '" . $lang['receivedfrom04'] . ":',
+  headers '" . $lang['receivedvia04'] . ":',
   id AS 'ID:',
-  headers AS 'Message Headers:',
-  from_address AS 'From:',
-  to_address AS 'To:',
-  subject AS 'Subject:',
-  size AS 'Size:',
+  headers as '" . $lang['msgheaders04'] . ":',
```

Comment:
all MySQL reserved words should be in uppercase (AS, CASE, WHEN, THEN, ELSE....)